src/bin/inx-chronicle/cli/mod.rs

```diff
@@ -335,8 +370,11 @@ pub enum Subcommands {
         /// Select a subset of analytics to compute.
         #[arg(long)]
         analytics: Vec<chronicle::db::influxdb::AnalyticsChoice>,
+        /// The input source to use for filling the analytics.
+        #[arg(long, value_name = "INPUT_SOURCE")]
+        input_source: InputSourceChoice,
```

Comment:
Default this

Reply:
Almost forgot this but luckily I didn't eventually. Thanks for reminding anyways.